api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -118,4 +116,30 @@
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
     
+	<changeSet id="20180214-1000" author="Kelechi+iheanyi">
+		<preConditions onFail="MARK_RAN">
+			<not><tableExists tableName="condition"/></not>
+		</preConditions>
+		<comment>
+			Add condition table
+		</comment>
+		<createTable tableName="condition">
+			<column name="condition_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="additional_detail" type="VARCHAR(255)">
+				<constraints nullable="true"/>
+			</column>
+			<column name="condition" type="VARCHAR(50)" defaultValue="null"/>
```

Comment:
Didn't we agree to lowercase VARCHAR?

Reply:
Yes we did. I'll have to ensure the change is effected.